be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -186,26 +186,33 @@ Status HdfsScanner::open_random_access_file() {
     CHECK(_file == nullptr) << "File has already been opened";
     ASSIGN_OR_RETURN(_raw_file, _scanner_params.fs->new_random_access_file(_scanner_params.path))
     _raw_file->set_size(_scanner_params.file_size);
-    auto stream = std::make_shared<CountedSeekableInputStream>(_raw_file->stream(), &_stats);
-    std::shared_ptr<io::SeekableInputStream> input_stream = stream;
+
+    std::shared_ptr<io::SeekableInputStream> input_stream = _raw_file->stream();
+    // if compression
+    // input_stream = DecompressInputStream(input_stream)
     if (_compression_type != CompressionTypePB::NO_COMPRESSION) {
         using DecompressorPtr = std::shared_ptr<StreamCompression>;
         std::unique_ptr<StreamCompression> dec;
         RETURN_IF_ERROR(StreamCompression::create_decompressor(_compression_type, &dec));
         auto compressed_input_stream =
-                std::make_shared<io::CompressedInputStream>(stream, DecompressorPtr(dec.release()));
+                std::make_shared<io::CompressedInputStream>(input_stream, DecompressorPtr(dec.release()));
         input_stream = std::make_shared<io::CompressedSeekableInputStream>(compressed_input_stream);
     }
 
+    // if block cache
+    // input_stream = CacheInputStream(input_stream)
     if (_scanner_params.use_block_cache && _compression_type == CompressionTypePB::NO_COMPRESSION) {
```

Comment:
so it still not support compression file?

Reply:
In theory it shoudl, but right now it's not well tested.